styles/style.css

```diff
@@ -0,0 +1,37 @@
+body{
```

Comment:
Across this file, we're missing space between the identifier we're targeting and the block of rules, best practices are to always leave a space before the curly brackets for visual clarity.
```css
body {
```